extensions/interactions/MultipleChoiceInput/webdriverio.js

```diff
@@ -43,12 +43,11 @@ var expectInteractionDetailsToMatch = async function(
   expect(optionsCount).toEqual(richTextInstructionsArray.length);
   var promises = [];
   for (var i = 0; i < optionsCount; i++) {
-    promises.push(await (await optionElements[i]).$(
-      '.e2e-test-multiple-choice-option')).getText();
+    promises.push(await optionElements[i].$(
+      '.e2e-test-multiple-choice-option').getText());
   }
   var rteInstructionArrayCopy = [...richTextInstructionsArray];
   rteInstructionArrayCopy.sort();
-  var results = await webdriverio.promise.all(promises);
```

Comment:
Why was this removed and also why was no error thrown?

Reply:
This line is added just to make sure all the promises are resolved but in webdriverio all the elments are already pushed using await (at line 46) to make sure all of them are resolved, hence we do not need this line.

Reply:
So, how is the results variable being referenced below?

Reply:
That was a mistake, rectified now. Thanks!

Reply:
Is this statement not working in webdriverio `await webdriverio.promise.all(promises);`?

Reply:
No, we do not have this command present in WebdriverIO.